app/controlplane/pkg/biz/organization.go

```diff
@@ -197,6 +198,32 @@ func (uc *OrganizationUseCase) FindByID(ctx context.Context, id string) (*Organi
 	return org, nil
 }
 
+// FindByName finds an organization by name.
+func (uc *OrganizationUseCase) FindByName(ctx context.Context, name string) (*Organization, error) {
```

Comment:
once you move the other code to biz, you might not need to expose this use-case nor the one below. You'll still need the repo though.